<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="selections">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>First</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Second</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Third</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Forth</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fifth</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_selections">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineedit_name"/>
    </item>
    <item>
     <widget class="QPushButton" name="button_name">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_description">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textbrowser_description"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="question">
      <property name="text">
       <string>Wanna add something?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plaintextedit_question"/>
    </item>
    <item>
     <widget class="QPushButton" name="button_question">
      <property name="text">
       <string>Submit and Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
